### PR TITLE
Add constant file and refactor

### PR DESCRIPTION
**Pull request by nturinski**
For after the release -- just wanted to have a better centralized constants file.  It's a wonder that it didn't already.

### CODE REVIEW

File: src/tree/VirtualMachineTreeItem.ts [hunk 1]
```diff
@@ -112,6 +113,9 @@ export class VirtualMachineTreeItem extends AzureTreeItem {
                 context.telemetry.properties.failedResources = failedResources.length.toString();
                 // if the vm failed to delete or was not being deleted, we want to throw an error to make sure that the node is not removed from the tree
                 if (failedResources.some(r => r.resourceType === virtualMachineLabel) || !context.deleteVm) {
+                    // tslint:disable-next-line: no-floating-promises
+                    const viewOutputAzureButton: AzExtErrorButton = { title: viewOutput.title, callback: async (): Promise<void> => ext.outputChannel.show() };
+                    context.errorHandling.buttons = [viewOutputAzureButton];
```
Comment (neelip) [inline src/tree/VirtualMachineTreeItem.ts:118]: What's the reason for this change?

Comment (nturinski) [reply src/tree/VirtualMachineTreeItem.ts:118]: This is behavior is already in main, I just refactored so that I wouldn't have to localize "View Output" twice.

What this does is when an error is thrown, it adds an additional button to the error message.  If a user clicks it, it invokes the callback.

Comment (ejizba) [reply src/tree/VirtualMachineTreeItem.ts:118]: Pretty sure there's just a merge conflict and if you merge/rebase with main, most of this diff will go away

File: src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as os from 'os';
+import { join } from "path";
+import { MessageItem } from "vscode";
+import { localize } from "./localize";
+
+export const viewOutput: MessageItem = { title: 'View Output' };
```
Comment (neelip) [inline src/constants.ts:11]: nit: localize

**Review by neelip — APPROVED** [review]

File: src/constants.ts (hunk 2 above)
Comment (nturinski) [reply src/constants.ts:11]:
```suggestion
export const viewOutput: MessageItem = { title: localize('viewOutput', 'View Output') };
```
